test/extended/prometheus/prometheus.go

```diff
@@ -181,6 +182,18 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			}
 			runQueries(tests, oc, ns, execPodName, url, bearerToken)
 		})
+		g.It("should be able to get the sdn ovs flows", func() {
```

Comment:
This test looks ok. However it is quite strange to have it as part of `[Feature:Prometheus][Conformance] Prometheus`.

Reply:
@smarterclayton suggested I put it here. Where would you suggest I put this instead?

Reply:
It's not wrong, it just looks a bit strange. Right now it is the easiest thing to do to just put it here otherwise it would probably need some code duplication. That said it:
/lgtm